dotCMS/src/main/java/com/dotmarketing/portlets/workflows/model/WorkflowAction.java

```diff
@@ -232,6 +233,16 @@ public void setCondition(String condition) {
 		this.condition = condition;
 	}
 
+	/**
+	 * Return true if the next step is the current step
+	 * @return boolean
+	 */
+	public boolean isNextStepCurrentStep() {
+
+		return null == this.getNextStep() ||
```

Comment:
do you need the null check here?  I don't think so.

Reply:
I think you right, we are taking care about the field on the factory, it shouldn't be null